src/dapp/Blockchain.ts

```diff
@@ -372,6 +374,52 @@ export class BlockChain {
     };
   }
 
+  private async getExploringAccount(exploringAccount): Promise<Account> {
+    if (!this.web3 || this.isTrial) {
```

Comment:
We can assume that if they were able to explore an account, that they are already connected to web3 and not in trial mode.

I would consider removing this check here.